Formula/abcm2ps.rb

```diff
@@ -20,7 +20,7 @@ def install
   end
 
   test do
-    (testpath/"voices.abc").write <<-EOF.undent
+    (testpath/"voices.abc").write <<~EOF
```

Comment:
@ilovezfs If you're feeling really pedantic I'd change these to `EOS` too but feel free to 👎 that.

Reply:
The set of all heredoc delimiters used in core is
EOS
'EOS'
XML
XSL
EOS_CONFIG
'EOS_CONFIG'
HTML
'EOF'
EOPLIST
EOI
'EOS_SRC'
EOM
EOCNF
PYTHON
PROTO
PLODRC
TEST_SCRIPT
ERR

Are there others besides `EOF` we also no longer :heart: ?

Reply:
@ilovezfs Personally I hate all non-EOS ones 😁. I think `'EOS'` has different behaviour, though, in terms of quoting etc. Will defer to you on them.

Reply:
I'm in favour of `<<~LANGUAGE` where it makes sense, because TextMate supports nested syntax highlighting this way, and other editors may too. `EOS` for the remaining cases.

Reply:
@reitermarkus which of them are taking advantage of that?

```
EOS
'EOS'
XML
XSL
EOS_CONFIG
'EOS_CONFIG'
HTML
EOF
'EOF'
EOPLIST
EOI
'EOS_SRC'
EOM
EOCNF
PYTHON
PROTO
PLODRC
TEST_SCRIPT
ERR
```

I'm guessing XML, XSL, HTML, and PYTHON? And maybe PROTO?

Reply:
I pushed an all-EOS commit https://github.com/Homebrew/homebrew-core/pull/19600/commits/7a81b5a0bedfc98bcb6ad2d0ce7aec25b94af41d but can consider backing some of those out.

Also, what is the magic delimiter for C? I assume we don't want to go down that road given how many C code examples we have.

Personally I think just standardizing on EOS is probably the simplest thing to do at least in formulae.

Reply:
>I'm guessing XML, XSL, HTML, and PYTHON? And maybe PROTO?

and maybe EOPLIST?

Reply:
> Also, what is the magic delimiter for C?

Just `C`.

> and maybe EOPLIST?

`plist` doesn't have its own, but `XML` works.

Reply:
Full list: `HTML`, `XML`, `CSS`, `SQL`, `CPP`, `C`, `JS|JAVASCRIPT`, `JQUERY`, `SH|SHELL`, `LUA`, `RUBY`,

Reply:
@reitermarkus for now I'd like to proceed with standardizing on EOS but we could consider supporting editor heredoc syntax highlighting at some point if that becomes a broadly supported feature in editors beyond TextMate. However, I think the value proposition is pretty low if the language has to be manually specified via the heredoc delimiter because it's currently something humans would need to police, as opposed to something that could be automatically enforced.